addons/l10n_fr/data/account_fiscal_position_template_data.xml

```diff
@@ -356,13 +311,13 @@
         <record id="fp_tax_template_impexp_ha_specifique" model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fiscal_position_template_import_export" />
             <field name="tax_src_id" ref="tva_acq_specifique" />
-            <field name="tax_dest_id" ref="tva_import_0" />
+            <field name="tax_dest_id" ref="tva_import_outside_ue_8_5" />
         </record>
 <!-- Taux Intermédiare -->
         <record id="fp_tax_template_impexp_ha_intermediaire" model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fiscal_position_template_import_export" />
             <field name="tax_src_id" ref="tva_acq_intermediaire" />
-            <field name="tax_dest_id" ref="tva_import_0" />
+            <field name="tax_dest_id" ref="tva_import_outside_ue_10" />
```

Comment:
@AdrienHorgnies Here, it is still ue

Reply:
Weird, I did a search and replace in the whole file

Reply:
Oh, I did only one of the two files, my bad. It's done now.